test/e2e/network/netpol/network_policy.go

```diff
@@ -1249,6 +1252,67 @@ var _ = common.SIGDescribe("Netpol [Feature:SCTPConnectivity][LinuxOnly][Disrupt
 	})
 })
 
+var _ = common.SIGDescribe("Netpol [Feature:NetworkPolicyEndPort][LinuxOnly]", func() {
+	f := framework.NewDefaultFramework("netpol-port-range")
+
+	ginkgo.BeforeEach(func() {
+		// should open the feature gate: NetworkPolicyEndPort
+		e2eskipper.SkipUnlessNetworkPolicyEndPortEnabled()
+	})
+
+	ginkgo.Context("NetworkPolicy between server and client using port range", func() {
+		ginkgo.BeforeEach(func() {
+			_, _, _, model, k8s := getK8SModelForEndPort(f)
+			initializeResourcesByFixedNS(k8s, model)
+		})
+
+		ginkgo.AfterEach(func() {
+			if !useFixedNamespaces {
+				_, _, _, model, k8s := getK8SModelForEndPort(f)
+				framework.ExpectNoError(k8s.deleteNamespaces(model.NamespaceNames), "unable to clean up UDP netpol namespaces")
+			}
+		})
+
+		ginkgo.It("should enforce policy to allow ingress traffic using port range [Feature:NetworkPolicy]", func() {
+			nsX, _, _, model, k8s := getK8SModelForEndPort(f)
+			ingressRule := networkingv1.NetworkPolicyIngressRule{}
+			endPort := int32(82)
```

Comment:
just an idea, .... how about a more obvious port range like say 111-115 ? that way its not confused with the 80,81,82 ports that we hardcode elsewhere ?

Reply:
done.